Readability.js

```diff
@@ -346,8 +346,9 @@ Readability.prototype = {
     }
 
     curTitle = curTitle.trim();
-
-    if (curTitle.split(' ').length <= 4)
+    var curTitleLen = curTitle.split(' ').length;
+    if (curTitleLen <=4 && ( !origTitle.match(/ [\\\/>»] /)
+        || curTitleLen != origTitle.replace(/[\|\-\\\/>» ]+/g, " ").split(' ').length -1))
```

Comment:
Thanks for the pull request! Certainly the first two additions to the regexps a bit higher up are fine. Here I'm less sure - can you explain what is going on with the second clause to the if statement?

Otherwise, two small style nits: we should probably use `.test()` rather than `.match()` for the first check here, and we normally have spaces around operators but not inside braces (`()`) (we need to put that in our eslint config if they aren't already, sorry if you ran that and this didn't come up. :-( ).